test/image/mocks/centered-tooltips.json

```diff
@@ -0,0 +1,58 @@
+{
+    "data": [
+        {
+            "y": [
+                1,
+                2,
+                6,
+                3,
+                5,
+                6
+            ],
+            "x": [
+                0,
+                1,
+                2,
+                3,
+                4,
+                5
+            ],
+            "text": [
+                "short label",
+                "short label",
+                "Very long label named San Francisco",
+                "Very long label named San Francisco",
+                "Very long label named San Francisco"
```

Comment:
Yep, this is one of the situations that trigger a centered label, when the text to be displayed is extra long. The other common one is a single-bar bar chart filling the whole plot area. I'm not sure we need a mock for this though, as image tests won't show hover labels - I think better would be to just generate these conditions as jasmine tests in https://github.com/plotly/plotly.js/blob/master/test/jasmine/tests/hover_label_test.js, and verify *something* about the label positioning/shape.